server/graphql/v2/object/Update.js

```diff
@@ -0,0 +1,118 @@
+    },
+    COLLECTIVE_ADMINS: {
+      description: 'Will be sent to collective admins',
+    },
+    FINANCIAL_CONTRIBUTORS: {
+      description: 'Will be sent to financial contributors',
+    },
+  },
+});
+
+const Update = new GraphQLObjectType({
+  name: 'Update',
+  description: 'This represents an Update',
+  fields: () => {
+    return {
+      id: {
+        type: new GraphQLNonNull(GraphQLString),
+        resolve: getIdEncodeResolver(IDENTIFIER_TYPES.UPDATE),
+      },
+      legacyId: { type: GraphQLInt },
+      slug: { type: new GraphQLNonNull(GraphQLString) },
+      userCanSeeUpdate: {
+        description: 'Indicates whether or not the user is allowed to see the content of this update',
+        type: new GraphQLNonNull(GraphQLBoolean),
+        resolve(update, _, req) {
+          if (!update.isPrivate) {
+            return true;
+          }
+          return req.remoteUser && req.remoteUser.canSeeUpdates(update.CollectiveId);
+        },
+      },
+      isPrivate: { type: new GraphQLNonNull(GraphQLBoolean) },
+      title: { type: new GraphQLNonNull(GraphQLString) },
+      createdAt: { type: new GraphQLNonNull(DateString) },
+      updatedAt: { type: new GraphQLNonNull(DateString) },
+      publishedAt: { type: DateString },
+      image: { type: GraphQLString },
+      views: { type: GraphQLInt },
+      notificationAudience: { type: UpdateAudienceTypeEnum },
+      makePublicOn: { type: GraphQLDateTime },
+      summary: {
+        type: GraphQLString,
+        resolve(update, _, req) {
+          if (update.isPrivate && !(req.remoteUser && req.remoteUser.canSeeUpdates(update.CollectiveId))) {
+            return null;
+          }
+
+          return update.summary || '';
+        },
+      },
+      html: {
+        type: GraphQLString,
+        resolve(update, _, req) {
+          if (update.isPrivate && !(req.remoteUser && req.remoteUser.canSeeUpdates(update.CollectiveId))) {
+            return null;
+          }
+
+          return stripHTML(update.html || '');
+        },
+      },
+      markdown: {
+        type: GraphQLString,
+        resolve(update, _, req) {
+          if (update.isPrivate && !(req.remoteUser && req.remoteUser.canSeeUpdates(update.CollectiveId))) {
+            return null;
+          }
+
+          return stripHTML(update.markdown || '');
+        },
+      },
+      tags: { type: new GraphQLList(GraphQLString) },
+      fromCollective: {
+        type: Account,
+        resolve(update, args, req) {
+          return req.loaders.Collective.byId.load(update.FromCollectiveId);
+        },
+      },
+      collective: {
+        type: Account,
+        resolve(update, args, req) {
+          return req.loaders.Collective.byId.load(update.CollectiveId);
+        },
+      },
+      tier: {
+        type: Tier,
+        resolve(update, args, req) {
+          return req.loaders.Tier.byId.load(update.TierId);
+        },
+      },
```

Comment:
This field is never set in the DB, looks like a feature that was never fully implemented. Let's not expose it

```suggestion
```